cmd/client-s3.go

```diff
@@ -196,7 +196,8 @@ func (n notifyExpiringTLS) RoundTrip(req *http.Request) (res *http.Response, err
 
 	cert := res.TLS.PeerCertificates[0] // leaf certificate
 	validityDur := cert.NotAfter.Sub(cert.NotBefore)
-	if time.Since(cert.NotBefore) > time.Duration(0.9*float64(validityDur)) {
+	// Warn if less than 10% of time left and it is less than 28 days.
+	if time.Since(cert.NotBefore) > time.Duration(min(0.9*float64(validityDur), 28*24*float64(time.Hour))) {
```

Comment:
I think you mean (validityDur-28*24*float64(time.Hour))

Reply:
Yeah, though changing it for clarity.